File: atcoder.jp/contests/exawizards2019/tasks/exawizards2019_b/main.cpp
#include <bits/stdc++.h>
using namespace std;

int count(char x, const string &s) { return count(s.begin(), s.end(), x); }

int main() {
  int N;
  string S;
  cin >> N >> S;

  bool ans = count('B', S) < count('R', S);
  cout << (ans ? "Yes" : "No") << endl;
}

File: atcoder.jp/contests/exawizards2019/tasks/exawizards2019_b/main.ts
import { readFileSync } from "fs";
let inputLinesIndex = 0;
const inputLines = readFileSync("/dev/stdin", "utf8").split("\n");
const input = () => inputLines[inputLinesIndex++];

const count: {
  (x: string, iterable: string): number;
  <T>(x: T, iterable: T[]): number;
} = <T>(x: string | T, iterable: string | T[]) =>
  typeof iterable === "string"
    ? iterable.split("").filter((a) => a === x).length
    : iterable.filter((a) => a === x).length;

const main = () => {
  input();
  const S = input();

  const ans = count("B", S) < count("R", S);
  console.log(ans ? "Yes" : "No");
};

main();
